code/modules/research/xenobiology/crossbreeding/chilling.dm

```diff
@@ -0,0 +1,634 @@
+			if(user.canUseTopic(src, BE_CLOSE) && user.canUseTopic(M, BE_CLOSE))
+				to_chat(user, "<span class='notice'>You store [M] in the capture device.</span>")
+				to_chat(M, "<span class='notice'>The world warps around you, and you're suddenly in an endless void, with a window to the outside floating in front of you.</span>")
+				store(M, user)
+			else
+				to_chat(user, "<span class='warning'>You were too far away from [M].</span>")
+				to_chat(M, "<span class='warning'>You were too far away from [user].</span>")
+		else
+			to_chat(user, "<span class='warning'>[M] refused to enter the device.</span>")
+			return
+	else
+		if(istype(M, /mob/living/simple_animal/hostile) && !("neutral" in M.faction))
+			to_chat(user, "<span class='warning'>This creature is too aggressive to capture.</span>")
+			return
+	to_chat(user, "<span class='notice'>You store [M] in the capture device.</span>")
+	store(M)
+
+/obj/item/capturedevice/attack_self(mob/user)
+	if(contents.len)
+		to_chat(user, "<span class='notice'>You open the capture device!</span>")
+		release()
+	else
+		to_chat(user, "<span class='warning'>The device is empty...</span>")
+
+/obj/item/capturedevice/proc/store(var/mob/living/M)
+	M.forceMove(src)
+
+/obj/item/capturedevice/proc/release()
+	for(var/atom/movable/M in contents)
+		M.forceMove(get_turf(loc))
+
+/obj/item/clothing/head/peaceflower
+	name = "heroine bud"
+	desc = "You feel at peace. How could you ever want to feel any different?"
+	icon = 'icons/obj/slimecrossing.dmi'
+	icon_state = "peaceflower"
+	item_state = "peaceflower"
+
+/obj/item/clothing/head/peaceflower/equipped(mob/living/carbon/human/user, slot)
+	. = ..()
+	if(slot == SLOT_HEAD)
+		user.add_trait(TRAIT_PACIFISM, "peaceflower_[REF(src)]")
+
+/obj/item/clothing/head/peaceflower/dropped(mob/living/carbon/human/user)
+	..()
+	user.remove_trait(TRAIT_PACIFISM, "peaceflower_[REF(src)]")
+
+/obj/item/clothing/head/peaceflower/attack_hand(mob/user)
+	if(iscarbon(user))
+		var/mob/living/carbon/C = user
+		if(src == C.head)
+			to_chat(user, "<span class='warning'>You feel at peace. <b style='color:pink'>Why would you want anything else?</b></span>")
+			return
+	return ..()
+
+/obj/item/clothing/suit/armor/heavy/adamantine
+	name = "adamantine armor"
+	desc = "A full suit of adamantine plate armor. Impressively resistant to damage, but weighs about as much as you do."
+	icon_state = "adamsuit"
+	item_state = "adamsuit"
+	flags_inv = list()
+	var/hit_reflect_chance = 40
+	slowdown = 4
+
+
+/obj/item/clothing/suit/armor/heavy/adamantine/IsReflect(def_zone)
+	if(def_zone in list(BODY_ZONE_CHEST, BODY_ZONE_R_ARM, BODY_ZONE_L_ARM, BODY_ZONE_R_LEG, BODY_ZONE_L_LEG) && prob(hit_reflect_chance))
+		return TRUE
+	else
+		return FALSE
+
+/obj/item/clothing/suit/armor/heavy/adamantine/Initialize()
+	. = ..()
+	START_PROCESSING(SSobj, src)
+
+/obj/item/clothing/suit/armor/heavy/adamantine/process()
+	slowdown = 4
```

Comment:
what

Reply:
I'm

I'm not sure what I was thinking at the time.